### ISSUE TITLE
Delete `decentraland-api` validation on `dcl start`

### ISSUE THREAD

**Issue by fmiras** [decentraland/cli#367]

**Comment by menduz** [comment]
isnt it required for XML scenes?

**Comment by fmiras** [comment]
yes but not the validation since we're not releasing `decentraland-api` new packages anymore

**Comment by menduz** [comment]
they may be using an outdated version anyway

**Comment by fmiras** [comment]
Yes, they could be running also `metaverse-api` but is another library that is also deprecated. Developers have the responsability of keep dependencies updated on their own, if they want keep using old SDK, they should keep using old CLI also. We should start supporting again this validation with `dcl-xml`.
